src/styles/blocks/catalog.scss

```
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
```

You should specify the property. 
for example: 
```suggestion
    transition: color 300ms;
```

---

But do it for all properties and use a variable for the duration

---

https://developer.mozilla.org/en-US/docs/Web/CSS/transition

---

And if you have a question, ask it in the fe_chat. It will be faster